core/src/Kokkos_Complex.hpp

```diff
@@ -720,7 +720,7 @@ KOKKOS_INLINE_FUNCTION Kokkos::complex<RealType> pow(const complex<RealType>& x,
   using std::pow;
   using std::sin;
 #endif
-  RealType phi = atan(x.imag() / x.real());
+  RealType phi = atan2(x.imag(), x.real());
```

Comment:
Don't you also have to import `sycl::atan2` instead of `sycl::atan` and `std::atan2` instead of `syd::atan`?

Reply:
Fixed.
Note that we should really be using the math functions from `Kokkos::Experimental::`